musical_chairs/lib/musical_chairs.ex

```diff
@@ -1,2 +1,47 @@
 defmodule MusicalChairs do
+  @input_file "input.txt"
+  @output_file "output.txt"
+  
+  def main(_args) do
+    play_musical_chairs
+  end
+
+  defp play_musical_chairs do
+    File.rm(@output_file)
+
+    @input_file
+    |> File.stream!
+    |> Stream.map(&line_to_children_and_chairs/1)
+    |> Stream.map(&play_round/1)
+    |> Stream.with_index
+    |> Enum.map(&write_round/1)
+  end
+
+  defp line_to_children_and_chairs(line) do
+    line
+    |> String.split
+    |> Enum.map(&String.to_integer/1)
+    |> List.to_tuple
+  end
+
+  defp play_round(round) do
```

Comment:
This function is redundant. You should be able to call `eliminated_children` directly.